### ISSUE TITLE
Nerpa/Calibration Network support

### ISSUE THREAD

**Issue by lanzafame** [textileio/bidbot#17]
Is it possible to test `bidbot` out on either nerpa or calibration networks, which I am guessing would require that there is a SB setup on either of those networks? Testing infrastructure configurations with sidecar applications in production is less than ideal.

**Comment by jsign** [comment]
The current version only connects to a SB that's targeting mainnet, yes. We thought about considering some test network our initial phase was mostly trying a 100% workable PoC with mainnet success; plus some initial early miner testers found inconvenient to run a miner in a test network.

As you mention, supporting a test network involves creating another SB environment that is using a Lotus gateway targeting the test network, so no big deal. From the bidbot side, maybe some extra flag to select the network (~SB) you want to connect to, and potentially doing some checks with the miner Lotus node to verify is connected to the right env. All quite doable, but sounds it also requires some satellite work to produce fake auctions and have a good UX for miners connecting there.

In short, is definitely a good idea and on the roadmap. In particular, entering the stage of massive miner onboarding.

**Comment by lanzafame** [comment]
Yeah, totally understand. 
> plus some initial early miner testers found inconvenient to run a miner in a test network.

I was actually going to suggest that nerpa be used for this reason, as it doesn't require a mainnet hardware setup so it allows for easier integration testing of all the infrastructure pieces, like `bidbot`, remote lotus nodes, monitoring, etc. Thoughts for the roadmap. Feel free to close this issue or leave it open so others don't ask as well.

**Comment by jsign** [comment]
Let's leave it open to gauge others wondering/wanting the same, and to discuss a bit when we start working on this.  :+1:
